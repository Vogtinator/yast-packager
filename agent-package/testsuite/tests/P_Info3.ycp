{
    return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"german","common.pkd":"common.short.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Execute(.setInstallSelection,["aaa_dir"],false);
}
{
    return SCR::Read(.packageList);
}
{
    return SCR::Read(.notifyDesc,"xxxxxxxxx");
}
