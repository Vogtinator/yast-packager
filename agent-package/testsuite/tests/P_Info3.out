(true)
(true)
($["aaa_base":["SuSE Linux Basispaket", "", 500, "2001.3.12-0"], "aaa_dir":["SuSE Linux Verzeichnisstruktur", "X", 912, "2001.1.17-0"], "aaa_skel":["SuSE Linux Default-Benutzer", "", 144, "2001.2.20-0"], "at":["Job-Manager", "", 70, "3.1.8-295"], "base":["Einige wichtige GNU-Programme", "", 1842, "2001.1.15-2"]])
("")
